.travis.yml

```diff
@@ -20,10 +20,10 @@ install:
   # PyKQML
   - pip install git+https://github.com/bgyori/pykqml.git
   # INDRA
-  - pip install git+https://github.com/sorgerlab/indra.git
+  - pip install git+https://github.com/sorgerlab/indra.git@kappy_integ
```

Comment:
Ah, this will be a problem, actually... maybe for the time being we need to duplicate the json processing code, just to not have to wait for merging the kappy_integ branch in indra.

Reply:
We could also merge the kappy_integ branch of indra. The errors seem to be a travis issue, not a substantial issue with the branch.